src/main/webapp/app/pages/curation/review/ReviewPage.tsx

```diff
@@ -40,20 +42,23 @@ const ReviewPage: React.FunctionComponent<IReviewPageProps> = (props: IReviewPag
   const [isReviewFinished, setIsReviewFinished] = useState(false);
 
   const [reviewUuids, setReviewUuids] = useState<string[]>(null);
-  const [rootReview, setRootReview] = useState<BaseReviewLevel>(undefined);
   const [reviewChildren, setReviewChildren] = useState<BaseReviewLevel[]>([]);
   const [editorReviewMap, setEditorReviewMap] = useState(new EditorReviewMap());
-
+  const [editorsToAcceptChangesFrom, setEditorsToAcceptChangesFrom] = useState<string[]>([]);
+  const [isAcceptingAll, setIsAcceptingAll] = useState(false);
   const [splitView, setSplitView] = useState(false);
 
+  const fetchFirebaseData = () => {
+    // Fetch the data when the user enters review mode. We don't use a listener
+    // because there shouldn't be another user editing the gene when it is being reviewed.
+    get(ref(props.firebaseDb, firebaseGenePath)).then(snapshot => setGeneData(snapshot.val()));
+    get(ref(props.firebaseDb, firebaseMetaReviewPath)).then(snapshot => setMetaReview(snapshot.val()));
+  };
+
   useEffect(() => {
```

Comment:
Probably a slight nit, but you could move the `props.GetDrugs` call to a separate `useEffect`. That way, you could save an API call or two if firebase is not yet ready